### PR TITLE
datadog-agent: 6.4.2 -> 6.8.3

### PR DESCRIPTION
**Pull request by Izorkin**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by memberbetty — APPROVED** [review]
I would like to see this in 18.09 too.

**Comment by Infinisil** [comment]
@GrahamcOfBorg build datadog-agent

**Comment by Izorkin** [comment]
In linux kernel 4.14 datadog v6.4.2 and v6.8.2 normal work.
In linux kernel 4.19 datadog v6.4.2 and v6.8.2 - error running check disk - https://github.com/DataDog/datadog-agent/issues/2822

**Comment by Izorkin** [comment]
Fixed check disk in this PR https://github.com/NixOS/nixpkgs/pull/53141

**Comment by Izorkin** [comment]
Update to 6.8.3

File: nixos/modules/services/monitoring/datadog-agent.nix [hunk 1]
```diff
@@ -186,7 +186,7 @@ in {
       type = types.attrs;
       default = {
         init_config = {};
-        instances = [ { use-mount = "no"; } ];
+        instances = [ { use-mount = "false"; } ];
```
Comment (Infinisil) [inline nixos/modules/services/monitoring/datadog-agent.nix]: What's this change for? Was the old value invalid?

Comment (Izorkin) [reply nixos/modules/services/monitoring/datadog-agent.nix]: This parameter found in example config https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/dist/conf.d/disk.d/conf.yaml.default#L6

**Comment by Izorkin** [comment]
Thanks!

**Pull request merged**